### ISSUE TITLE
When printing a process object a message appears saying to "refresh" the object

### ISSUE THREAD

**Issue by nighttardis** [carbonblack/carbon-black-cloud-sdk-python#100]
**I am seeing this behaviour on: (please complete the following information):**
 - OS: ubuntu/popos 20.04
 - Carbon Black Cloud Products: Enterprise EDR
 - Python Version: 3.8

**Describe the bug**
When running printing a process object a message is also printed that the object is "Partially initialized. Use .refresh() to load all attributes". Except the process object is an extension of the "UnrefreshableModel" class so refresh won't work on it.

**Steps to Reproduce**
```python
query = queryAPI.select(Process).where("process_name:cmd.exe")
for p in query:
    print(p)
    p.refresh()
```

**Expected behavior**
Logic to handle these "UnrefreshableModel" classes as to lessen the confusion on if "refresh" is needed or not.

**Screenshots**
If applicable, add screenshots or code to help explain your problem.

**Additional context**

**Comment by avanbrunt-cb** [comment]
PR created: https://github.com/carbonblack/carbon-black-cloud-sdk-python/pull/104

**Comment by avanbrunt-cb** [comment]
Fixed in v1.1
